source/dev_guide/dev_guide_index.rst

```diff
@@ -41,6 +41,12 @@ General
 
       /dev_guide/ace_file.rst
 
+   .. toctree::
+      :maxdepth: 1
+      :glob:
+
+      /dev_guide/dev/dev_guide_localization.rst
```

Comment:
I dont know who added all the .rst extensions, but they are not needed in the doctree directive, also all the weird seperate toctree directives are also not needed, probably dont hurt but not needed.